src/bot.js

```diff
@@ -84,7 +84,7 @@ client.on("message", async message => {
   if (!prefix.length) prefix = config.prefix;
 
   if (message.content.startsWith(prefix) || message.content.match(`^<@!?${client.user.id}> `)) return commandHandler(message, gdb, db, channel, prefix);
-  else if (channel == message.channel.id) return countingHandler(message, gdb); // TODO add args
+  else if (channel == message.channel.id && message.type === "DEFAULT") return countingHandler(message, gdb); // TODO add args
```

Comment:
I'd like this to rather be in the if-check in line 73.

Reply:
Done.